package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Jymfony",
   "main": "index.js",
   "scripts": {
-    "test": "node ./tests.js \"src/*/test/**/*.js\" \"src/Component/*/test/**/*.js\"",
-    "coverage": "istanbul cover ./tests.js \"src/*/test/**/*.js\" \"src/Component/*/test/**/*.js\""
+    "test": "node ./tests.js \"src/*/test/**/*.js\" \"src/Component/*/test/**/*.js\" \"src/Bundle/*/test/**/*.js\"",
```

Comment:
Comma separated names should be a valid glob. Please use `src/{Bundle,Component}/*/test/**/*.js` instead of adding a new path.